### PR TITLE
Added PHP-HttpRequest2 codegen

### PR DESCRIPTION
**Pull request by shreys7**
Added PHP-HttpRequest2 Codegen
Docs: https://pear.php.net/manual/en/package.http.http-request2.intro.php

### CODE REVIEW

File: codegens/php-httprequest2/test/newman/newman.test.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+var runNewmanTest = require('../../../../test/codegen/newman/newmanTestUtil').runNewmanTest,
+  convert = require('../../index').convert;
+
+
+describe('PHP HttpRequest2 Converter', function () {
+  // Skipping newman tests for now
+  // TODO: Addition of set_include_path() in generated snippet
+  describe.skip('convert for different request types', function () {
```
Comment (umeshp7) [inline codegens/php-httprequest2/test/newman/newman.test.js]: We can remove this file and npm/test-newman.js for now if we are not planning to add newman tests right now.

Comment (shreys7) [reply codegens/php-httprequest2/test/newman/newman.test.js]: We can keep it for future reference, as I have added a TODO to note what is left to be added.

**Comment by umeshp7** [comment]
Also, add this code generator in the list of codegen in root README.md.

**Comment by shreys7** [comment]
> Also, add this code generator in the list of code-generators in root README.md. @shreys7

@umeshp7 done.

**Pull request merged**